puppet/modules/kubernetes/manifests/proxy.pp

```diff
@@ -23,6 +24,14 @@
     ensure => 'present',
   })
 
+  if $::kubernetes::proxy_feature_gates == [] {
+    $_feature_gates = delete_undef_values([
+      $::kubernetes::_enable_pod_priority ? { true => 'PodPriority=true', default => undef },
```

Comment:
I don't think this is needed. Only needed for API server, scheduler and kubelet https://v1-10.docs.kubernetes.io/docs/concepts/configuration/pod-priority-preemption/